.github/workflows/deploy.yml

```diff
@@ -52,8 +52,8 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: npm run build
         env:
+          NODE_ENV: "production"
```

Comment:
Not doubting this, but what was the reason to add this?

Reply:
I think the various nested `?:` operators happen to work either way right now, but being explicit about building everything for the public site.

https://github.com/pantsbuild/pantsbuild.org/blob/main/docusaurus.config.js#L20